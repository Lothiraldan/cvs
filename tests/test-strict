#!/bin/sh

hg init

echo a > a
hg ci -d '0 0' -Ama

hg an a

echo "[ui]" >> $HGRCPATH
echo "strict=True" >> $HGRCPATH

hg an a
hg annotate a

echo % should succeed - up is an alias, not an abbreviation

hg up
