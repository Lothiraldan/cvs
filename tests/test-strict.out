adding a
0: a
hg: unknown command 'an'
Mercurial Distributed SCM

basic commands:

 add        add the specified files on the next commit
 annotate   show changeset information per file line
 clone      make a copy of an existing repository
 commit     commit the specified files or all outstanding changes
 diff       diff repository (or selected files)
 export     dump the header and diffs for one or more changesets
 init       create a new repository in the given directory
 log        show revision history of entire repository or files
 merge      merge working directory with another revision
 parents    show the parents of the working directory or revision
 pull       pull changes from the specified source
 push       push changes to the specified destination
 remove     remove the specified files on the next commit
 serve      export the repository via HTTP
 status     show changed files in the working directory
 update     update working directory

use "hg help" for the full list of commands or "hg -v" for details
0: a
% should succeed - up is an alias, not an abbreviation
0 files updated, 0 files merged, 0 files removed, 0 files unresolved
